Utils/Extensions/StringExtensions.swift

```diff
@@ -52,6 +52,31 @@ public extension String {
         return str as! String
     }
 
+    /**
+    Truncates a String in the middle only if it's longer than `maxLength`
+
+      "ABCDEF".truncate(4)
+      // "AB…EF"
+
+    :param: maxLength The maximum length of the String.
+
+    :returns: A truncated String
+    */
+    func truncate(#maxLength: Int) -> String {
+        if count(self) >= maxLength {
```

Comment:
Should be `>`, otherwise you'll ellipsize a string that's the maximum valid length.